apps/hash-ai-worker-ts/src/activities/user-exceeded-service-usage-limit-activity.ts

```diff
@@ -11,7 +11,7 @@ const usageCostLimit = {
     month: 150,
   },
   user: {
-    day: 10,
+    day: 50,
```

Comment:
Did you mean to commit this?

Reply:
Nope, thanks for catching - have removed in https://github.com/hashintel/hash/pull/4497/commits/47b3eb7cda7ab4364bf7f4448e4200336defc117